.github/workflows/eamxx_default_files.yml

```diff
@@ -0,0 +1,72 @@
+name: inputdata
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: '00 00 * * *'
+  workflow_dispatch:
+
+jobs:
+  scream-defaults:
+    runs-on: ubuntu-latest
+    outputs:
+      event_name: ${{ github.event_name }}
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v4
+        with:
+          show-progress: false
+          submodules: false
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4.7.0
+        with:
+          python-version: "3.11"
+      - name: Run unit tests
+        working-directory: components/eamxx/cime_config/
+        run: |
+          python -m unittest tests/eamxx_default_files.py -v
+
+  notify-scream-defaults:
+    needs: scream-defaults
+    if: failure() && needs.scream-defaults.outputs.event_name == 'pull_request'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create issue
+        run: |
+          if [[ $CLOSE_PREVIOUS == true ]]; then
```

Comment:
This seems overly fancy... Why not simply appending to the existing issue?

Reply:
I mean, you can still check if the issue is open, and if so, append to it. I just think we don't need to close the old issue and open a new one...